nni/algorithms/compression/__init__.py

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import logging
+
+_logger = logging.getLogger(__name__)
+_logger.warning('import path `nni.algorithms.compression` has be removed in v3.0, please import from `nni.compression`.')
```

Comment:
has been

Reply:
good point, just want to give more information for different import levels, if raise error here, it will directly raise error in the upper level, for example:
```
>>> from nni.algorithms.compression.v2.pytorch.pruning import xx
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/xxx/nni/nni/algorithms/compression/__init__.py", line 4, in <module>
    raise RuntimeError('something')
RuntimeError: something
```
I can't tell user change `from nni.algorithms.compression.v2.pytorch.pruning import xx` to `from nni.compression.pytorch.pruning import xx` by raise error.

Reply:
What I did in NAS is that I kept all the original files but deleted their contents. Added a shortcut to the new location.

Reply:
align with NAS